react/components/InputSearch/index.js

```diff
@@ -73,15 +75,19 @@ class InputSearch extends Component {
         onBlur={() => this.handleFocus(false)}
         onMouseEnter={() => this.handleHovering(true)}
         onMouseLeave={() => this.handleHovering(false)}
-        onKeyUp={e => e.key === 'Enter' && this.handleSubmit(e)}
+        onKeyUp={e => this.pressedEnter(e) && this.handleSubmit(e)}
```

Comment:
do you know why is the `KeyUp` event being used here instead of `KeyPress`?

Reply:
no, I don't. I just realized that now. Don't `onKeyUp` has better performance if the user keeps the same key pressed?

Reply:
depends on the use case :sweat_smile: I was just wondering if you had an idea on why this was used instead of `onKeyPress`, don't really need to change anything